lib/deck.rb

```diff
@@ -0,0 +1,19 @@
+class Deck
+    attr_accessor :cards
+
+    def initialize(cards) 
+        @cards = cards
+    end 
+
+    def count
+        @cards.count 
+    end 
+
+    def cards_in_category(specific_category)
+        @cards.select{|card| card.category == specific_category}
+    end   
+    
+    def categories
```

Comment:
🔥 I haven't looked at your `Round` class yet but I have a strong hunch about what this method is used for and I'm very happy to see it. This allows your program to be very dynamic which is pretty much always a good thing.